### ISSUE TITLE
Ability to have multiple collection files (276 Votes)

### ISSUE THREAD

**Issue by gaddlord** [gaddlord/mtg.studio#332]
This could work both ways, first is a way to create backups of the current collection and also will be able to manage collections from other people (we have different collections but pull them on when we create decks) and thus know who owns what

Votes: 276 by 123 supporters
id: 191182
Created: 15.05.2009 19:02 by anonymous ()
Updated: 08.10.2021 21:45

**Comment by gaddlord** [comment]
Daniel LeGrand:
Plus 1 for this idea. Would love to be able to separate my MTGO and paper cards into unrelated collections.

**Comment by gaddlord** [comment]
Jared:
I would love to be able to store my collection in a custom location..ie dropxbox so I can access it from another machine.

**Comment by gaddlord** [comment]
Joshua Brown:
I'd like to see this feature added since I play in a couple of weekly leagues and need to keep the collections separate of one another entirely.

**Comment by gaddlord** [comment]
labadie:
Now that MTGO is available to be listed in the database, being able to update the online side would be a WHOLE lot easier if it was separate from the paper listing.

**Comment by gaddlord** [comment]
Mjavado:
This indeed is a feature I'm missing. I do suggest an option to view the collections together. I borrow cards from my girlfriend and she does so from me, and keeping track of it by making a separate collection of borrowed cards could be useful. At the same time you'll want to see the cards as yours in your overall inventory (being all collections).

**Comment by gaddlord** [comment]
Matt Coon:
I'd like two things. first to store my "primary" collection in "My Documents" and second to change my collection as i share files with friends for trading.

**Comment by gaddlord** [comment]
Mary:
This would be a wonderful addition for someone like me: I'm a dealer & also a collector, so I have two very separate collections: The first is my "for sale" binders/boxes, which is constantly changing, and the second is my "personal" collection, which is periodically added to. To mix the two into one listed collection would be folly, and extremely unhelpful. I need to be able to see exactly what I have to sell, as well as exactly what I have personally, allowing me to tell how many cards I need from a certain set/artist/creature type etc., without counting the cards I have for sale in that tally. Looking forward to this!!

**Comment by gaddlord** [comment]
Rob:
make it happen

**Comment by gaddlord** [comment]
Gad:
Commands New, Open, Save, Save As, Close to work also for Collections.

**Comment by gaddlord** [comment]
GeekDoc:
I would greatly appreciate this feature. I would then be able to inventory my children's collections.

**Comment by gaddlord** [comment]
cougarcrossing:
This will allow us to trade via email, etc.

**Comment by gaddlord** [comment]
zorbaelbudha:
I don't know if this is possible now, but I will like to have the ability to use several inventory files, so I could track different inventories (as I am for now the caretaker of the collections of a couple of friends of mine and want to keep them separately).

Also the option to be able to backup your inventories could be useful as I normally try to have a couple of backups if my main inventory file becomes corrupted.

**Comment by gaddlord** [comment]
Schwanke:
The easiest way to do multiple collections is two stage.

The collection file is read on running of the program and saved on exit so basically you want to copy the one you want to work with into the current file and copy out the one you want to save elsewhere. Its mtgstudio.collection for the installed version and mtgstudio_portable.collection for the portable version.

The other option which is what I do is when I want to load a collection I just delete everything in thec urrent collection and then import the collectoin I want to work with.

To save it I use the same copy it out of the saved file.

Theoretically you could use batch files if you really wanted to to automate the process but I was too lazy last time to do that. Might do that in the future if I need to handle more then 5 or 10. Gets tedious though writing the batch files and checking them for small errors so they don't eat your whole collection

**Comment by gaddlord** [comment]
Anvil:
Hey all!

What I've done is whenever I close MTG Studio, I renamed the "mtgstudio.collection" file to something else. For instance, I have a collector's album of the Mirrodin block. So when I entered what was in the album, I renamed that entire file to "Mirrodin Album.collection" after closing MTG Studio. I find that whenever you do this, MTG Studio will recreate an empty "mtgstudio.collection" when opened again and not even read my renamed collection file. BUT, I can import my renamed file just fine. The only problem with THAT is that the import wizard asks to either append or merge the imported collection file. There is no option to clear it or open it separately (like having multiple documents open in MS Word or Libreoffice). So if I'm working on a different collection and I want to add something to my Mirrodin album file, I would have to close MTG Studio, rename or delete the mtgstudio.collection file (or clear out the collection I was working on), import my "Mirrodin Album.collection" file, modify the collection (add cards / remove cards /etc.), delete the old "Mirrodin Album.collection" file, and rename the created mtgstudio.collection to Mirrodin Album.collection. That's how I do it.

I think this will all be handled better once gaddlord is able to get to the New, Open, Save, Save As and Close features for collections. That is how Magic Workstation did it, and it would be even more awesome in MTG Studio.

Again, the way I do it in steps:
1. Open MTG Studio and create your collection as normal.
2. When done, close MTG Studio.
3. Go to the location where mtgstudio.collection file is created (usually in Windows 7 it's "C:\Users\YourUserName\AppData\Roaming\MTG Studio").
4. Rename the file to something that you'll recognize. ( CollectionName.collection )

Remember that once you rename the file, MTG Studio will automatically create mtgstudio.collection when you start the program. This will be a pain when you go to modify your collection when you're already working on another one.

To modify collection:
1. Open MTG Studio.
2. Import your renamed file. MAKE SURE YOU AREN'T WORKING ON SOMETHING, OR RENAME "mtgstudio.collection" FILE IF YOU ARE! Otherwise, you will lose whatever you are doing currently.
3. Make your changes.
4. Close MTG Studio.
5. If you had deleted or renamed "mtgstudio.collection", MTG Studio will create this file whenever it is opened and the file doesn't already exist. So now this is what your modified collection will be. You can confirm by checking the "Date modified" column in Windows Explorer.
6. Delete your original collection and rename this new/recent "mtgstudio.collection" to your collection name.

Hope all this helps. At least until file manipulation comes out in the newer versions.

**Comment by gaddlord** [comment]
CptDobey:
Another way to do it is to install the portable version multiple times on different folders.
As an example:
c:\program files\MtgStudio
c:\program files\\MtgStudio_MTGO
c:\program files\\MtgStudio_Trade
...

I put the "card art" into c:\program files\MtgStudio and set up the configuration of c:\program files\\MtgStudio_MTGO to look into c:\program files\\MtgStudio\cart art instead of its default one. This way, I do not need to copy image files multiple times. You can probably delete the "My decks" directories in all but one installation folders as well.

You just have to create shortcuts to the differents .EXE files et voilà. You have different collections without the need to rename/delete/...

Apparently, you are not able to run multiple instance of MTG Studio at once, but maybe there is a flag somewhere to allow it. I didn't search for it ;)

Anyway, I have a question: why don't you filter you collection with F3 (upper grid) or shift-F3 (for the lower grid) instead of creating multiple files (per collection) ?

**Comment by gaddlord** [comment]
Anvil:
Hello! It's been a while since I've touched or played with MTG Studio. In fact, after I built my new PC, I had forgotten what the program was called, as I wanted to take another look at it, and it took me 20 minutes on Google to find it. Has there been any improvements in the "opening multiple inventory files" front?

**Comment by gaddlord** [comment]
mark pinkston:
the ability to track foils separately so the price would be accurate would be invaluable

**Comment by gaddlord** [comment]
Azul:
When you import your .csv file from MTGO it should create a new collection called MTGO Collection. Then you can create a deck from your regular collection or create a deck from your MTGO collection.

For me this would be more useful because I have a ton of different cards on MTGO than I do in paper. The decks I make in MTGO are going to be different than the ones I can make in paper because I have a totally different online collection.

**Comment by gaddlord** [comment]
thothgirl:
Is the ability to have multiple collections on the list for updates? It's not as important as decks from collection, but would still be a nice feature. just wondering what, if any, priority this feature has or would have.

**Comment by gaddlord** [comment]
Schwanke: My main use idea is that I want to be able to manage multiple lists of cards with different grades and keep track of the prices of sell and then the auto lookup of price as well.

**Comment by gaddlord** [comment]
Ouronos: You could also have different Inventories to use and choose from.

**Comment by gaddlord** [comment]
Mystiqu: I think this application focuses too much on card features and misses the bigger picture.

I would like to have a view with a list of all your decks (or your favorite decks) so you have a fast and easy access to them and can switch between them easily (one click only to load).
To go into the menu and load a new deck every time you want to switch to another one is very annoying and you have no clear overview of all your decks.

Just a simple view/work space to the left would do - like:
My Decks View
*Green infect deck
*White control deck
*Red bomber
*...

The same goes for the collection, which today is a big junk yard.
I would like to have a list of different collections as well, like:

*My entire collection
*Items I want to trade away
*Wish list
*... and so on...

So, to summarize:
1) Editable deck view with your favorite decks
2) Editable collection view with all your collections

**Comment by gaddlord** [comment]
TheAdrian: Yeah sounds good. Especially the ability to have multiple collections etc. I'm starting to wish the program would have a visual approach for collections and decks a bit like mtgo; I find mtgo collection view and deck builder very clumsy but I'm sure a more elegant version could be done in mtgstudio.

For multiple collections an icon for each by double clicking opens up the collection which can be displayed either like in the current form like a spreadsheet or a visual card collection and similarly for the deck editor.

Furthermore once a collection has been opened up one could have further icons for each set/expansion/block visually.

Obviously the collection/deck editor currently in mtgstudio is the core mechanics of the program which can be at some point (as an option) be more fancy visually.

**Comment by gaddlord** [comment]
Mystuqu: This is a great suggestion.
Under each collection symbol you could have symbols for all the sets/expansions/blocks.
One click on the collection symbol displays all the cards in the collection, and then a specific set is just a click away :D

A feature like this is really easy to implement since the core functionality behind it most likely is already in place.

**Comment by oboobortot** [comment]
Delivered — the web app supports unlimited independent named collections (create/rename/delete), each with its own card list, stats, and value; they're server-backed so they double as cloud backups. Closing as delivered.
